frontend/utils.mjs

```diff
@@ -129,6 +129,7 @@ export const commonConfig = {
         'process.env.NODE_ENV': isDev ? '"development"' : '"production"',
     },
     loader: {
+        '.ttf': 'file',
```

Comment:
What's this for?

Reply:
Got an error when importing from monaco otherwise :/